### PR TITLE
Lps 152440 6 16

### PR DESCRIPTION
**Pull request by slnn**

### CODE REVIEW

**Comment by tinatian** [comment]
@vicnate5 Please help me scan test results.

Thanks.

**Comment by vicnate5** [comment]
@tinatian No regressions found.

**Comment by dantewang** [comment]
@slnn Conflict on bnd.bnd again. You can just update and force push to avoid re-running CI.

**Comment by slnn** [comment]
@dantewang 

I have update and force pushed.

**Comment by tinatian** [comment]
@slnn Code conflicts, please rebase.

Tina.

**Comment by slnn** [comment]
Hi @tinatian 

Update and force pushed.

/cc @dantewang 

Lily

**Comment by slnn** [comment]
Hi @dantewang @tinatian 

bnd conflict again, update and force pushed.

**Comment by tinatian** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/1068

File: portal-kernel/src/com/liferay/portal/kernel/templateparser/TemplateNode.java [hunk 1]
```diff
@@ -209,13 +208,9 @@ public String getUrl() {
...
 		String data = (String)get("data");
 
-		JSONObject jsonObject = JSONUtil.createJSONObject(data);
-
-		if (jsonObject == null) {
-			return StringPool.BLANK;
-		}
-
 		try {
+			JSONObject jsonObject = JSONFactoryUtil.createJSONObject(data);
```
Comment (tinatian) [inline portal-kernel/src/com/liferay/portal/kernel/templateparser/TemplateNode.java:212]: @slnn I did not include this change, as this is going to generate an exception when jsonObject is null, it is better that we keep the original checking.

**Pull request closed without merging**